Mathlib/Computability/NFA.lean

```diff
@@ -112,6 +120,275 @@ theorem pumping_lemma [Fintype σ] {x : List α} (hx : x ∈ M.accepts)
   rw [← toDFA_correct] at hx ⊢
   exact M.toDFA.pumping_lemma hx hlen
 
+section Closure
+
+/-! ### NFA Closure Properties -/
```

Comment:
It would be good to add a short doc comment along the lines of:

> Properties of the form "if these languages can be represented by NFAs, then these related languages also can".

This is to avoid confusion with e.g. Kleene's closure.

Reply:
Ok, sounds good. I added something but I am not sure of the best way to phrase it nor if the documentation formatting/style is good.